### PR TITLE
Hide projects and lists

### PR DESCRIPTION
**Pull request by jtourkos**
Here is the expected behaviour: https://github.com/drips-network/app/issues/1242#issuecomment-2421895623

### CODE REVIEW

File: src/lib/components/drip-list-card/drip-list-card.svelte [hunk 1]
```diff
@@ -226,6 +228,7 @@
   href={dripListUrl}
   target={openInNewTab ? '_blank' : undefined}
   class="drip-list-card rounded-drip-lg overflow-hidden shadow-low group"
+  class:hidden-list={isHidden}
```
Comment (mhgbrown) [inline src/lib/components/drip-list-card/drip-list-card.svelte:231]: You have `hidden-list` and a `hiddenList` class above. How about keeping it kebab case?

File: src/lib/components/drip-lists-section/drip-lists-section.svelte [hunk 2]
```diff
@@ -107,12 +125,31 @@
...
     </div>
+
+    <div>
+      {#if isOwner}
+        <VisibilityToggle bind:showHidden hiddenItemsCount={hiddenListsCount} />
+      {/if}
+    </div>
+
+    <div
+      class="grid gap-6 grid-cols-1 padding pt-px {hiddenDripListsAndVotingRounds.length > 0
```
Comment (mhgbrown) [inline src/lib/components/drip-lists-section/drip-lists-section.svelte:144]: Wish I was this fluent with the helper classes 🥲

File: src/lib/components/identity-card/identity-card.svelte [hunk 3]
```diff
@@ -84,10 +87,20 @@
       <div class="icon">
         <DripListIcon style="fill: var(--color-primary); height: 3rem; width: 3rem;" />
       </div>
-      <span class="typo-header-3 ellipsis">{dripList.name}</span>
+
+      <div>
+        <span class="typo-header-3 ellipsis" class:hiddenByUser={!dripList.isVisible}
```
Comment (mhgbrown) [inline src/lib/components/identity-card/identity-card.svelte:92]: Another camel case class btw

Comment (mhgbrown) [reply src/lib/components/identity-card/identity-card.svelte:92]: ...and some more sprinkled throughout the PR.

**Review by mhgbrown — COMMENTED** [review]
Noice. I couldn't notice anything about it initially, but are featured projects and drip lists also covered by the `isVisible` logic here? (Do they even need to be covered?) `FEATURED_DRIP_LISTS` is a constant...could those become hidden?

File: src/lib/components/drip-list-card/drip-list-card.svelte (hunk 1 above)
Comment (jtourkos) [reply src/lib/components/drip-list-card/drip-list-card.svelte:231]: 👍

File: src/lib/components/drip-lists-section/drip-lists-section.svelte (hunk 2 above)
Comment (jtourkos) [reply src/lib/components/drip-lists-section/drip-lists-section.svelte:144]: all copy-paste magic  😇

File: src/lib/components/identity-card/identity-card.svelte (hunk 3 above)
Comment (jtourkos) [reply src/lib/components/identity-card/identity-card.svelte:92]: 👍

File: src/lib/components/project-card/project-card.svelte [hunk 4]
```diff
@@ -12,9 +13,15 @@
       chainData {
         ... on ClaimedProjectData {
           chain
+          owner {
+            accountId
+          }
```
[the remark this answers is not in the document]
Comment (jtourkos) [reply src/lib/components/project-card/project-card.svelte:18]: It's required for checks like `$: isOwner = $walletStore.connected && checkIsUser(projects[0]?.chainData[0]?.owner?.accountId);`

**Pull request merged**